MatrixMultiplication/MatrixMultiplication/PerformanceReport.cs

```diff
@@ -0,0 +1,110 @@
+using System.Diagnostics;
+using Aspose.Pdf;
+using Aspose.Pdf.Text;
+
+namespace MatrixMultiplication;
+
+public static class PerformanceReport
+{
+    private const int Trials = 5;
+    private static readonly Stopwatch Stopwatch = new Stopwatch();
```

Comment:
```suggestion
    private static readonly Stopwatch Stopwatch = new();
```
Не пишите один тип дважды